eng/pipelines/templates/jobs/archetype-sdk-client.yml

```
@@ -121,6 +121,29 @@ stages:
         displayName: "Show Docker Logs"
         condition: succeededOrFailed()
 
+    - ${{ if startsWith(parameters.ServiceDirectory, 'resourcemanager') }}:
+      - job: MgmtMockTest
+        displayName: Mock Test For Mgmt Plane
+        variables:
+          - template: ../variables/globals.yml
```

```suggestion
          - template: /eng/pipelines/templates/variables/globals.yml
```
Better for these templates to use full path so that they don't break when things move around.